tests/test_python.py

```diff
@@ -109,21 +110,36 @@ def test_tox(auto_container):
     auto_container.connection.run_expect([0], "pip install --user tox")
 
 
-def test_python_devel(auto_container):
-    """Check that python_devel package can be installed."""
+def test_cryptography(auto_container):
```

Comment:
The `auto_container` fixture is intended for tests that only read as all tests share the same container.

```suggestion
def test_cryptography(auto_container_per_test):
```

Reply:
I guess you don't only want to change in the definition but also in the body of the test function? fixed.

Reply:
yes